docs/TerraformImportPlugin.md

```diff
@@ -0,0 +1,29 @@
+## [TerraformImportPlugin](../src/TerraformImportPlugin.groovy)
+
+Enable this plugin to change pipeline functionality. This plugin will import a
+resource into state, and adds two new job parameters.
+
+* `IMPORT_RESOURCE`: This is the resource identifier to import into state.
+* `IMPORT_TARGET_PATH`: This is the Terraform state path into which the
+  resource should be imported.
```

Comment:
There should be a third `IMPORT_ENVIRONMENT` here right?

Reply:
Missed that... added.